src/pk.c

```diff
@@ -9788,12 +9787,12 @@ char* wolfSSL_EC_POINT_point2hex(const WOLFSSL_EC_GROUP* group,
              * odd.
              */
             hex[0] = mp_isodd((mp_int*)point->Y->internal) ?
-                ECC_POINT_COMP_ODD : ECC_POINT_COMP_EVEN;
+                0x03 : 0x02;
```

Comment:
Did you switch to using the numeric directly to avoid having the `#fndef HAVE_SELFTEST`?

I really prefer the `ECC_POINT_COMP_ODD`, `ECC_POINT_COMP_EVEN` and `ECC_POINT_UNCOMP` values. If needed you could add some logic above to ensure these are available.

```
#ifdef HAVE_SELFTEST
    /* point compression types */
    #define ECC_POINT_COMP_EVEN 0x02
    #define ECC_POINT_COMP_ODD  0x03
    #define ECC_POINT_UNCOMP    0x04
#endif
```